<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>466</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <widget class="QWidget" name="widget_10" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>233</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 900 36pt &quot;Montserrat&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; color:#5663b6;&quot;&gt;Handy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 900 36pt &quot;Montserrat&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#5663b6;&quot;&gt;Match&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>271</width>
          <height>221</height>
         </rect>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>0</y>
           <width>211</width>
           <height>211</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="MainWindow.qrc">:/prefijoNuevo/Logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <zorder>widget</zorder>
       <zorder>widget_10</zorder>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_9" native="true">
       <widget class="QWidget" name="FondoInisesion" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>271</width>
          <height>391</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:15;
background-color: rgb(115, 134, 244);</string>
        </property>
        <widget class="QPushButton" name="pushButtonLogin">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>210</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 15px;
    color: rgb(255, 255, 255);
    font: 900 9pt &quot;Montserrat&quot;;
    background-color: rgb(255, 137, 33);
    border: 3px solid #ff8921;
    selection-background-color: rgb(186, 100, 47);
}
QPushButton:hover {
    background-color: rgb(255, 160, 70);
    border: 3px solid #ffa94d; 
}
</string>
         </property>
         <property name="text">
          <string>Iniciar sesión</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>280</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 300 9pt &quot;Montserrat&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;No tiene cuenta?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonRegistro">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>300</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: rgb(190, 190, 190);
    font: 600 9pt &quot;Montserrat&quot;;
    selection-color: rgb(186, 100, 47);
}
QPushButton:hover {
   color: rgb(220, 220, 220);
}
</string>
         </property>
         <property name="text">
          <string>Registrese</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>260</y>
           <width>231</width>
           <height>2</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: #5663b6;
    max-height: 2px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
        </widget>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>251</width>
           <height>171</height>
          </rect>
         </property>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>120</y>
            <width>271</width>
            <height>41</height>
           </rect>
          </property>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>51</height>
            </rect>
           </property>
           <widget class="QLineEdit" name="lineEditContrasena">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>0</y>
              <width>191</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;
border-right-color: rgb(120, 140, 255);
border-right: 3px solid #5663b6;
border-top:3px solid #5663b6;
border-bottom:3px solid #5663b6;
background-color: rgb(115, 134, 244);
color: white;
</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::InputMethodHint::ImhHiddenText</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Contraseña</string>
            </property>
           </widget>
           <widget class="QLabel" name="Candado">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>0</y>
              <width>41</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-right:none;
background-color: rgb(115, 134, 244);
border-left: 3px solid #5663b6;
border-top: 3px solid #5663b6;
border-bottom: 3px solid #5663b6;


</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="MainWindow.qrc">:/prefijoNuevo/cerrar.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>261</width>
            <height>61</height>
           </rect>
          </property>
          <widget class="QLineEdit" name="lineEditEmail">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>10</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Montserrat</family>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;
border-right-color: rgb(120, 140, 255);
border-right: 3px solid #5663b6;
border-top:3px solid #5663b6;
border-bottom:3px solid #5663b6;
background-color: rgb(115, 134, 244);
color: white;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Correo electronico</string>
           </property>
          </widget>
          <widget class="QWidget" name="Correo_2" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>41</width>
             <height>41</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-right:none;
background-color: rgb(115, 134, 244);
border-left: 3px solid #5663b6;
border-top: 3px solid #5663b6;
border-bottom: 3px solid #5663b6;


</string>
           </property>
           <widget class="QLabel" name="Correo">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>41</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-right:none;
background-color: rgb(115, 134, 244);
border-left: 3px solid #5663b6;
border-top: 3px solid #5663b6;
border-bottom: 3px solid #5663b6;


</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="MainWindow.qrc">:/prefijoNuevo/sobre.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QLabel" name="Labelsesion">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>10</y>
            <width>261</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 900 18pt &quot;Montserrat&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Iniciar sesion&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>251</width>
           <height>161</height>
          </rect>
         </property>
        </widget>
        <zorder>widget_7</zorder>
        <zorder>widget_6</zorder>
        <zorder>pushButtonLogin</zorder>
        <zorder>label_2</zorder>
        <zorder>pushButtonRegistro</zorder>
        <zorder>frame</zorder>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
